library/src/compute/calc.rs

```diff
@@ -270,6 +271,27 @@ pub fn atan(
     Value::Angle(Angle::rad(value.float().atan()))
 }
 
+/// Calculate the four-quadrant arctangent of a coordinate.
+///
+/// ## Example
+/// ```example
+/// #calc.atan(1, 1) \
+/// #calc.atan(-2, -3)
```

Comment:
```suggestion
/// #calc.atan2(1, 1) \
/// #calc.atan2(-2, -3)
```

Reply:
Weird, I thought I fixed that. Must have messed up the git stuff. Anyway, fixed now.